src/components/course.html

```diff
@@ -0,0 +1,65 @@
+<script defer src="https://use.fontawesome.com/releases/v5.15.4/js/all.js" integrity="sha384-rOA1PnstxnOBLzCLMcre8ybwbTmemjzdNlILg8O7z1lUkLXozs4DHonlDtnE7fpc" crossorigin="anonymous"></script>
+
+
+<div class="main-course">
+    <span>Learn from our 3 in 1 Professional Course</span>
+    <h3>Become a Certified Real Estate Consultant</h3>
+    <div class="course-group">
+        <div class="course-item">
+            <figure>
+                <img src="../assets/img/certified/Rectangle 12.png" alt="">
+            </figure>
+            <p>The Art of Selling</p>
+        </div>
+        <div class="course-symbol">
+            <b>4.0</b>
+            <i class="fa-sharp fa-solid fa-star"></i>
+            <i class="fa-sharp fa-solid fa-star"></i>
+            <i class="fa-sharp fa-solid fa-star"></i>
+            <i class="fa-sharp fa-solid fa-star"></i>
+            <span>(5,908)</span>
+        </div>
+        <div class="course-btn">
+            <button class="btn-courseitem">View course</button>
+        </div>
+    </div>
+    <div class="course-group">
+        <div class="course-item">
+            <figure>
+                <img src="../assets/img/certified/Rectangle 12.png" alt="">
+            </figure>
+            <p>The Art of Selling</p>
+        </div>
+        <div class="course-symbol">
+            <b>4.0</b>
+            <i class="fa-sharp fa-solid fa-star"></i>
```

Comment:
if rating is some other number will you rewrite the html code? I think you should find a way when changing the rating, the number of stars will show up correctly.